### PR TITLE
[Sprint 5] Edit Profile (Fragment to Activity)

### PR DESCRIPTION
**Pull request by Tuanlvt**
Task: Profile
Link redmine: https://dev.framgia.com/redmine/issues/138406

### CODE REVIEW

File: app/src/main/java/com/framgia/fbook/screen/profile/ProfileActivity.kt [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+import com.fstyle.library.MaterialDialog
+import com.fstyle.structure_android.widget.dialog.DialogManager
+import javax.inject.Inject
+
+/**
+ * Profile Screen.
+ */
+class ProfileActivity : BaseActivity(), ProfileContract.ViewModel {
+
+  private val PAGE_LIMIT = 1
```
Comment (HyperionsThs) [inline app/src/main/java/com/framgia/fbook/screen/profile/ProfileActivity.kt]: move to companion object pls

File: app/src/main/java/com/framgia/fbook/screen/profile/ProfileActivity.kt [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+  @Inject
+  internal lateinit var presenter: ProfileContract.Presenter
```
Comment (HyperionsThs) [inline app/src/main/java/com/framgia/fbook/screen/profile/ProfileActivity.kt]: change presenter to  mPresenter

File: app/src/main/java/com/framgia/fbook/screen/profile/ProfileAdapter.kt [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+import android.support.v4.app.Fragment
+import android.support.v4.app.FragmentManager
+import android.support.v4.app.FragmentPagerAdapter
+import com.framgia.fbook.R
+import com.framgia.fbook.screen.menuprofile.MenuProfileContainerFragment
+
+/**
+ * Created by levutantuan on 9/21/17.
+ */
+class ProfileAdapter(private val context: Context,
```
Comment (HyperionsThs) [inline app/src/main/java/com/framgia/fbook/screen/profile/ProfileAdapter.kt]: change context to mContext

File: app/src/main/java/com/framgia/fbook/screen/profile/ProfileModule.kt [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+import com.fstyle.structure_android.widget.dialog.DialogManagerImpl
+import dagger.Module
+import dagger.Provides
+
+/**
+ * This is a Dagger module. We use this to pass in the View dependency to
+ * the {@link ProfilePresenter}.
+ */
+@Module
+class ProfileModule(private val activity: Activity) {
```
Comment (HyperionsThs) [inline app/src/main/java/com/framgia/fbook/screen/profile/ProfileModule.kt]: change activity to mActivity

File: app/src/main/java/com/framgia/fbook/screen/profile/ProfileContainerFragment.kt [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    fun newInstance(position: Int): ProfileContainerFragment {
+      var profileContainerFragment: ProfileContainerFragment = ProfileContainerFragment()
+      var bundle: Bundle = Bundle()
+      bundle.putInt(EXTRA_TAB, position)
+      profileContainerFragment.arguments = bundle
+      return profileContainerFragment
+    }
+  }
+
+  private val mNavigator: Navigator by lazy { Navigator(this) }
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/fbook/screen/profile/ProfileContainerFragment.kt]: pls move top Top of file

**Pull request merged**